### ISSUE TITLE
Relay access denied

### ISSUE THREAD

**Issue by thomas126** [mailcow/mailcow-dockerized#1933]
I've git a running mailcow instance working fine. Just added another domain to it and set all dns settings for it forwarding mx to the main domain where the server is running. I also set an alias of domain b mail account forwarding to domain a account. When I sent an email form an external server to the domain b mail address it should be send to domain a server forwarding the mail to the domain a mail address. I am receiving the following error:

 Recipient address: mail@domain.b
 Reason: Remote SMTP server has rejected address
 Diagnostic code: smtp;554 5.7.1 <mail@domain.b>: Relay access denied
 Remote system: dns;mail.domain.a (TCP|17.172.80.98|46347|104.248.42.190|25) (mail.domain.a ESMTP Postfix)

Is that a dns error or where is the problem?

**Comment by mitchellurgero** [comment]
What do you mean forward? Are you forwarding from an email account or did you setup domain forwarding in the admin panel? You should not be relaying openly.

What are you trying to do?

**Comment by thomas126** [comment]
All setup in mailbox itself. I have added two domains in the webinterface. And one mail address of domain b is alias of domain a. Frankly mails send to that alias are being rejected...

**Comment by mitchellurgero** [comment]
That's strange, I'll try to replicate the issue on my end, in the mean time can you send any more logs?

**Comment by thomas126** [comment]
postfix-mailcow_1    | Oct 23 12:01:39 mail postfix/smtpd[18919]: connect from st11p00im-asmtp004.me.com[17.172.80.98]
postfix-mailcow_1    | Oct 23 12:01:39 mail postfix/smtpd[18919]: Anonymous TLS connection established from st11p00im-asmtp004.me.com[17.172.80.98]: TLSv1.2 with cipher ECDHE-RSA-AES128-GCM-SHA256 (128/128 bits)
postfix-mailcow_1    | Oct 23 12:01:39 mail postfix/smtpd[18919]: NOQUEUE: reject: RCPT from st11p00im-asmtp004.me.com[17.172.80.98]: 554 5.7.1 <mail@domain.b>: Relay access denied; from=<mail@icloud.com> to=<mail@domain.b> proto=ESMTP helo=<st11p00im-asmtp004.me.com>
postfix-mailcow_1    | Oct 23 12:01:39 mail postfix/smtpd[18919]: disconnect from st11p00im-asmtp004.me.com[17.172.80.98] ehlo=2 starttls=1 mail=1 rcpt=0/1 data=0/1 quit=1 commands=5/7

**Comment by andryyy** [comment]
I don't quite understand. Can you post the output of this? =>

```
source mailcow.conf ; docker-compose exec mysql-mailcow mysql -u${DBUSER} -p${DBPASS} ${DBNAME} -e "select * from domain; select * from alias_domain;"
```

**Comment by mitchellurgero** [comment]
How are you using icloud.com as a domain in Mailcow? You do not own that domain.

**Comment by andryyy** [comment]
Yes, good catch, didn't see that.

**Comment by thomas126** [comment]
I am not. What I am doing is sending an email from my iCloud account to mailcow. There it gets denied. Even though the addresser is an alias of a mailbox within Mailcow...

Interesting is that I am able to send a mail from within mailcow to the same adress (the alias) which is then correctly forwarded!

**Comment by andryyy** [comment]
Can you please still post the output of the command above? I don't understand your text in the first post at all. :-/

**Comment by mitchellurgero** [comment]
> What I am doing is sending an email from my iCloud account to mailcow. There it gets denied. Even though the addresser is an alias of a mailbox within Mailcow

Oh, That's not what was explained before, but that's fine. Now that I know this information I can test more since it's an external -> internal forward not an internal <-> internal forward.

**Comment by thomas126** [comment]
Sorry that I didn’t clear it up in the first place. So let me try again. I have mailcow running on a server with the domain mail.test.de. There is a mail account running named mail@test.de. That one is working fine. I also setup another domain test.me and created an alias called mail@test.me which is supposed to forward all mails to the first mailbox: mail@test.de. From within mailcow it’s working fine. When trying to send an email from another mail service to mail@test.me it gets denied by the server.

Is that now clearer? Thanks for all your help!

**Comment by mitchellurgero** [comment]
Create a dummy mail account for test.me and see if that helps.

**Comment by thomas126** [comment]
okay so I tried with a dummy account. The mail still gets rejected...

**Comment by mitchellurgero** [comment]
Try removing both the mailbox, and the alias and readd them, mailbox first. Once the alias is created, login as the mailbox the alias is assigned to and confirm you can send as the alias. Once you can confirm that (Done in SOGo settings I believe) then send the alias an email.

**Comment by thomas126** [comment]
I already did delete the alias. There was an error. Its not possible to have an alias setup together with a mail box using the name name. It set it up within mailcow settings web ui not sogo...

**Comment by mitchellurgero** [comment]
OK Do this:

Make new mailbox: mailbox1@domaina.com
THEN make an alias: alias1@domaina.com (Point to mailbox@domainb.com)

**Comment by mitchellurgero** [comment]
Mailbox name and alias name CANNOT be the same just FYI.

**Comment by thomas126** [comment]
still not working. I really have no idea what is going on. I also tried to send me another mail to a third domain which is working just fine. Checked the dns settings and they are basically the same so it should work too...

**Comment by thomas126** [comment]
Wow sorry guys. I just checked all settings in mailcow again because I didnt understand the error and I noticed the domain was set as inactive. So sorry for all the misunderstandings! But thanks for all your help.

**Comment by andryyy** [comment]
Asked two times about the SQL data, still no info... I cannot help you then.

**Comment by andryyy** [comment]
That's exactly why we needed the SQL data. It prints backupmx (that was my guess) and the active state.

**Comment by thomas126** [comment]
sorry again andryyy. Thanks for your help anyway!
